core/src/test/java/edu/wpi/grip/core/http/HttpPipelineSwitcherTest.java

```diff
@@ -79,7 +78,7 @@ public void open(String projectXml) {
         didRun[0] = true;
       }
     };
-    pipelineSwitcher = new HttpPipelineSwitcher(new ContextStore(), project, GripMode.HEADLESS);
+    pipelineSwitcher = new HttpPipelineSwitcher(new ContextStore(), project);
```

Comment:
Move this constructor into a helper method in the test so you don't have to change all usages of it